src/components/Loans/OpenLoans/components/OpenLoansWithStaticData/OpenLoansWithStaticData.js

```diff
@@ -176,13 +176,17 @@ class OpenLoansWithStaticData extends React.Component {
     const { resources } = this.props;
     const accounts = get(resources, ['loanAccount', 'records'], []);
     const accountsLoan = accounts.filter(a => a.loanId === loan.id) || [];
+    let suspendedStatus = '';
     let remaining = 0;
 
+    accountsLoan.forEach(a => {
+      suspendedStatus = (a.paymentStatus.name === 'Suspended claim returned') ? '\nSuspended' : '';
+    });
```

Comment:
`Suspended` needs to be localized, i.e. passed through `<FormattedMessage id="ui-users..." />`. Also, it's not clear to my why the `\n` is there, unless this is displayed inside a `<pre>` block.

Reply:
@zburke We add `FormattedMessage`